azure-pipelines.yml

```diff
@@ -66,7 +66,7 @@ stages:
 
           Write-Host "Remove .Net 5 target for UnitTest project"
 
-          ((Get-Content -Path $testProjFileName -raw) -Replace "net48;netcoreapp3.1;net5.0","net48;netcoreapp3.1") | Set-Content -Path $testProjFileName
+          ((Get-Content -Path $testProjFileName -raw) -Replace "net48;net5.0","net48") | Set-Content -Path $testProjFileName
```

Comment:
Do we still need this entire step `Exclude net5 from the main build`? It was introduced during .NET 5 preview times. Back then, net48 and netcoreapp3.1 were build. So can we just remove the step now?

Reply:
No it cannot be removed at this moment. `VSBuild` Azure task currently fails when building projects targeting .Net 5.